src/engine/libengine/PhoneticSuggestion.cpp

```diff
@@ -21,8 +21,10 @@
 
 PhoneticSuggestion::PhoneticSuggestion() {
   // Set patterns and optimize them
+
+  // Using RAW String literal (C++11)
   rgxPadding.setPattern(
-      "(^(?::`|\\.`|[-\\]\\\\~!@#&*()_=+\\[{}'\";<>/?|.,])*?(?=(?:,{2,}))|^(?::`|\\.`|[-\\]\\\\~!@#&*()_=+\\[{}'\";<>/?|.,])*)(.*?(?:,,)*)((?::`|\\.`|[-\\]\\\\~!@#&*()_=+\\[{}'\";<>/?|.,])*$)");
+      R"((^(?::`|\.`|[-\]\\~!@#&*()_=+\[{}'";<>/?|.,])*?(?=(?:,{2,}))|^(?::`|\.`|[-\]\\~!@#&*()_=+\[{}'";<>/?|.,])*)(.*?(?:,,)*)((?::`|\.`|[-\]\\~!@#&*()_=+\[{}'";<>/?|.,])*$))");
```

Comment:
Why an extra bracket in Regex `R"((^`? Previously it was `"(^`.

Reply:
@mominul  Details : https://www.geeksforgeeks.org/raw-string-literal-c/